tests/console/yast2_tftp.pm

```diff
@@ -55,14 +55,17 @@ sub run {
 
     # view log
     send_key 'alt-v';                                                                     # open log window
-    assert_screen([qw(yast2_tftp_view_log_error yast2_tftp_view_log_show)]);
+    assert_screen('yast2_tftp_view_log_show');
+    send_key 'alt-c';                                                                     # close the window
+    assert_screen([qw(yast2_tftp_view_log_error yast2_tftp_closed_port)]);
     if (match_has_tag('yast2_tftp_view_log_error')) {
         # softfail for opensuse when error for view log throws out
         record_soft_failure "bsc#1008493";
-        send_key 'alt-o';                                                                 # confirm the error message
+        wait_screen_change { send_key 'alt-o' };                                          # confirm the error message
+        assert_screen('yast2_tftp_view_log_show');
```

Comment:
My original intention was to solve it by a cool needle only, not any code change. A wait_screen_change before an assert_screen does not look something that we want to have in general. Why do you think this will help here?

Reply:
I actually thought that in this case code executed in this order :
```
wait_screen_change
send_key
assert_screen
```

@okurz ?

Reply:
@asmorodskyi no, you got it wrong. `wait_screen_change` ensures to wait for a screen change after the body was executed. I crosschecked the docs in http://open.qa/api/testapi/#_wait_screen_change and it is not exactly clear from that. I will propose a doc change

Reply:
@okurz fine.